### PR TITLE
Increment the banner_etc_issue_net rule

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

This rule is also applicable to RHEL products. So, RHEL products were included on the `prodtype` list.
In addition, the following improvements were made in this PR:
- CCEs for RHEL were included
- RHEL7 control file was updated
- Ansible remediation was created

#### Rationale:

Improve CIS coverage

### CODE REVIEW

**Comment by Mab879** [comment]
Can you please take a look at the Automatus test failures? I'm getting similar results when running Automatus locally on a VM.

**Comment by marcusburghardt** [comment]
> Can you please take a look at the Automatus test failures? I'm getting similar results when running Automatus locally on a VM.

I found the issue and it will be fixed soon. I will also update the test scenario scripts to cover the same cases of `banner_etc_issue_net`.

**Comment by marcusburghardt** [comment]
I rebased this PR and included more commits to fix the automatus tests. More details are in the commit messages.
In short, the tests scenarios which have the `stig` profile defined in their headers were not remediating the system because the `banner_etc_issue_net` rule was not included in any `stig` profile.

I simply removed the respective test scenario scripts since the same quality tests are already covered by other scripts, just with a different content. In the future, if this rule becomes part of `stig`, we can easily copy the test scenarios scripts from `banner_etc_issue` rule with small changes.

**Comment by Mab879** [comment]
So it looks like Ubuntu 20.04 does use the rule in the STIG. Maybe we add a filter only to run these tests in Ubuntu 20.04?

```
$ grep -ir banner_etc_issue_net products/*/profiles controls 
products/ubuntu2004/profiles/stig.profile:    - banner_etc_issue_net
controls/cis_rhel8.yml:      - banner_etc_issue_net
controls/cis_rhel7.yml:      - banner_etc_issue_net
```

**Comment by marcusburghardt** [comment]
> So it looks like Ubuntu 20.04 does use the rule in the STIG. Maybe we add a filter only to run these tests in Ubuntu 20.04?
> 
> ```
> $ grep -ir banner_etc_issue_net products/*/profiles controls 
> products/ubuntu2004/profiles/stig.profile:    - banner_etc_issue_net
> controls/cis_rhel8.yml:      - banner_etc_issue_net
> controls/cis_rhel7.yml:      - banner_etc_issue_net
> ```

Good find. I believe we can do it. Let me check.

**Comment by marcusburghardt** [comment]
Updated

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/bash/shared.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_ubuntu
+# platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/bash/shared.sh:1]:
```suggestion
# platform = multi_platform_all
```

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/ansible/shared.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/ansible/shared.yml:1]:
```suggestion
# platform = multi_platform_all
```

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_with_extra_line.fail.sh [hunk 3]
```diff
@@ -1,30 +1,30 @@
 #!/bin/bash
-#
+# platform = multi_platform_ubuntu
 # profiles = xccdf_org.ssgproject.content_profile_stig
 
 # dod_default|dod_short banner
-echo "You are accessing a U.S. Government (USG) Information System (IS) that is
-provided for USG-authorized use only. By using this IS (which includes any
+echo "You are accessing a U.S. Government (USG) Information System (IS) that is
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_with_extra_line.fail.sh:6]: Do we want these extra spaces at the end of the line?

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_dod_default_banner.pass.sh [hunk 4]
```diff
@@ -1,29 +1,29 @@
 #!/bin/bash
-#
+# platform = multi_platform_ubuntu
 # profiles = xccdf_org.ssgproject.content_profile_stig
 
 # dod_default banner
-echo "You are accessing a U.S. Government (USG) Information System (IS) that is
-provided for USG-authorized use only. By using this IS (which includes any
+echo "You are accessing a U.S. Government (USG) Information System (IS) that is
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_dod_default_banner.pass.sh:6]: Was it intentional to have these spaces at the end of the lines?

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/bash/shared.sh (hunk 1 above)
Comment (marcusburghardt) [reply linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/bash/shared.sh:1]: Good. I will include in the next update.

Comment (marcusburghardt) [reply linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/bash/shared.sh:1]: Good. I will include in the next update.

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/ansible/shared.yml (hunk 2 above)
Comment (marcusburghardt) [reply linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/ansible/shared.yml:1]: Good. I will include in the next update.

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_dod_default_banner.pass.sh (hunk 4 above)
Comment (marcusburghardt) [reply linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_dod_default_banner.pass.sh]: They were not intentional. They were actually inherited from `banner_etc_issue` rule. I will fix there too in order avoid this issue be replicated in the future by copy and paste.

**Review by Mab879 — DISMISSED** [review]

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_usgcb_banner.fail.sh [hunk 5]
```diff
@@ -1,29 +1,12 @@
...
-to protect USG interests--not for your personal benefit or privacy.
-
--Notwithstanding the above, using this IS does not constitute consent to PM, LE
-or CI investigative searching or monitoring of the content of privileged
-communications, or work product, related to personal representation or services
-by attorneys, psychotherapists, or clergy, and their assistants. Such
-communications and work product are private and confidential. See User
-Agreement for details. I've read & consent to terms in IS user agreem't." > /etc/issue.net
+# usgcb_default banner
+echo "-- WARNING -- This system is for the use of authorized users only. Individuals
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_usgcb_banner.fail.sh:6]: That banner doesn't seem to match the Ubuntu STIG see https://stigaview.com/products/ubuntu2004/v1r6/UBTU-20-010002/

Comment (marcusburghardt) [reply linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_usgcb_banner.fail.sh]: Done.

**Review by Mab879 — APPROVED** [review]
All Automatus tests locally on RHEL 7.9, 8.6, and 9.0.

**Pull request merged**